docs/docs/launch-manual/sections/using-pulsar/sections/editing-and-deleting-text.md

```diff
@@ -0,0 +1,243 @@
+### Editing and Deleting Text
+
+So far we've looked at a number of ways to move around and select regions of a
+file, so now let's actually change some of that text. Obviously you can type in
+order to insert characters, but there are also a number of ways to delete and
+manipulate text that could come in handy.
+
+#### Basic Manipulation
+
+There are a handful of cool keybindings for basic text manipulation that might
+come in handy. These range from moving around lines of text and duplicating
+lines to changing the case.
+
+::: tabs#using-pulsar
+
+@tab Linux
+
+- <kbd>Ctrl+J</kbd> - Join the next line to the end of the current line
+- <kbd>Ctrl+Up/Down</kbd> - Move the current line up or down
+- <kbd>Ctrl+Shift+D</kbd> - Duplicate the current line
+- <kbd>Ctrl+K</kbd> <kbd>Ctrl+U</kbd> - Upper case the current word
+- <kbd>Ctrl+K</kbd> <kbd>Ctrl+L</kbd> - Lower case the current word
+
+@tab macOS
+
+- <kbd>Cmd+J</kbd> - Join the next line to the end of the current line
+- <kbd>Cmd+Ctrl+Up/Down</kbd> - Move the current line up or down
+- <kbd>Cmd+Shift+D</kbd> - Duplicate the current line
+- <kbd>Cmd+K</kbd> <kbd>Cmd+U</kbd> - Upper case the current word
+- <kbd>Cmd+K</kbd> <kbd>Cmd+L</kbd> - Lower case the current word
+- <kbd>Ctrl+T</kbd> - Transpose characters. This swaps the two characters on either side of the cursor.
+
+@tab Windows
+
+- <kbd>Ctrl+J</kbd> - Join the next line to the end of the current line
+- <kbd>Ctrl+Up/Down</kbd> - Move the current line up or down
+- <kbd>Ctrl+Shift+D</kbd> - Duplicate the current line
+- <kbd>Ctrl+K</kbd> <kbd>Ctrl+U</kbd> - Upper case the current word
+- <kbd>Ctrl+K</kbd> <kbd>Ctrl+L</kbd> - Lower case the current word
+
+:::
+
+Pulsar also has built in functionality to re-flow a paragraph to hard-wrap at a
+given maximum line length. You can format the current selection to have lines no
+longer than 80 (or whatever number `editor.preferredLineLength` is set to)
+characters using
+**_LNX/WIN_**: <kbd>Alt+Ctrl+Q</kbd> -
```

Comment:
```suggestion
**_LNX/WIN_**: <kbd>Ctrl+Shift+Q</kbd> -
```

from https://github.com/pulsar-edit/pulsar-edit.github.io/pull/85

Reply:
Thanks for the correction